miniwallet/deploy/001_deploy_miniWallet.ts

```diff
@@ -0,0 +1,85 @@
+import config from '../config'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { DeployFunction } from 'hardhat-deploy/types'
+import { ethers } from 'hardhat'
+
+const OPERATOR_ROLE = ethers.utils.id('OPERATOR_ROLE')
+
+const deployFunction: DeployFunction = async function (
+  hre: HardhatRuntimeEnvironment
+) {
+  const { deployments, getNamedAccounts, getChainId } = hre
+  const { deploy } = deployments
+  const { deployer } = await getNamedAccounts()
+  const chainId = await getChainId()
+  let initialOperatorThreshold
+  let initialOperators
+  let initialUserLimit
+  let initialAuthLimit
+
+  console.log(`chainId: ${chainId}`)
+  if (chainId === '1666600000,') {
+    console.log('Harmony Mainnet Deploy')
+    initialOperatorThreshold = config.mainnet.miniWallet.initialOperatorThreshold
+    initialOperators = config.mainnet.miniWallet.initialOperators
+    initialUserLimit = config.mainnet.miniWallet.initialUserLimit
+    initialAuthLimit = config.mainnet.miniWallet.initialAuthLimit
+  } else {
+    console.log(`Test Deploy on chainId: ${chainId}`)
+    initialOperatorThreshold = config.test.miniWallet.initialOperatorThreshold
+    initialOperators = config.test.miniWallet.initialOperators
+    initialUserLimit = config.test.miniWallet.initialUserLimit
+    initialAuthLimit = config.test.miniWallet.initialAuthLimit
+  }
+
+  console.log('operators:', JSON.stringify(initialOperators))
+  const deployedContract = await deploy('MiniWallet', {
```

Comment:
Where can we control and store deployed proxy address, storage address, and logic contract address?